assets/sass/v2/_enroll-phone.scss

```diff
@@ -2,11 +2,6 @@
   .okta-form-subtitle {
     text-align: center;
   }
-  .chzn-container {
-    // TODO: Find a way around important
-    /* stylelint-disable declaration-no-important */
-    width: 100% !important;
-  }
```

Comment:
Looks like this is coming from '.o-form-wide' now. Please update the PR with screenshots for Voice Enrollment as well.

Reply:
yeah it is an option in courage inputs. 

![Screen Shot 2020-09-02 at 9 18 54 PM](https://user-images.githubusercontent.com/17267130/92070846-041feb00-ed62-11ea-9107-6712b3abae36.png)